frontend/src/streamlist/__tests__/StreamsTableWrapper.test.jsx

```diff
@@ -15,30 +15,47 @@ describe("<StreamsTableWrapper/> Component", () => {
       return Promise.resolve([]);
     }
   };
+  const dummyColumns = [
+    {
+      title: "ID",
+      field: "id"
+    },
+    {
+      title: "Date",
+      field: "date"
+    }
+  ];
 
-  describe("Should contain the following components", () => {
-    beforeEach(() => {
-      wrapper = Enzyme.shallow(
-        <StreamsTableWrapper dataSource={dummySource} />
-      );
-    });
+  beforeEach(() => {
+    wrapper = Enzyme.shallow(
+      <StreamsTableWrapper dataSource={dummySource} columns={dummyColumns} />
+    );
+  });
+
+  afterEach(() => {
+    wrapper.unmount();
+  });
+
+  describe("Should contain the following component", () => {
     it("Contains 1 <StreamsTable/> component", () => {
       expect(wrapper.find(StreamsTable)).toHaveLength(1);
     });
+    describe("<StreamsTable/> props", () => {
+      it("Columns should passed <StreamsTableWrapper/>'s columns prop", () => {
```

Comment:
since you're writing these tests. Try to make it in the same format as our new tests
```
it ("Contains 1 <ComponentName/> component with expected props", ()=>{
  const components = wrapper.find(ComponentName);
  expect(components).toHaveLength(1);
  
  const props = components.at(0).props();
  const expected={
    prop: expectedvalue
    ....
  }
  expect (props).toStrictEqual(expected)
})
```